envoy/datadog_checks/envoy/metrics.py

```diff
@@ -10,11 +10,17 @@
     'envoy_cluster_assignment_timeout_received': 'cluster.assignment_timeout_received',
     'envoy_cluster_bind_errors': 'cluster.bind_errors',
     'envoy_cluster_default_total_match_count': 'cluster.default_total_match',
+    'envoy_cluster_health_check_degraded' : 'cluster.health_check_degraded',
```

Comment:
Nice! Can you also add these metrics to [metadata.csv](https://github.com/DataDog/integrations-core/blob/master/envoy/metadata.csv) and make sure they are [included in testing](https://github.com/DataDog/integrations-core/blob/master/envoy/tests/common.py#L21)?